Tests/smokeTest.js

```diff
@@ -0,0 +1,41 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
+var fashionCategory = require ("../Pages/fashionCategoryPage");
+var itemPage = require ("../Pages/itemPage");
+var dataJS = require("../Pages/data");
+
+describe ('001: Smoke test', function(){
+    browser.waitForAngularEnabled(false);
+    browser.ignoreSynchronization = true;
+    browser.manage().window().maximize();
+    
+    it ("001: User is able to sign in, click on item and place a bid", function (){
+        homepage.open(dataJS.homepageLink)
+        .then(() => browser.wait(dataJS.EC.presenceOf($(dataJS.headerText)), 5000))
+        .then(() => homepage.clickOnLogin())
+        .then(() => browser.wait(dataJS.EC.presenceOf($(dataJS.emailLocator)), 5000))
+        .then(() => loginpage.enterEmail(dataJS.userRafaNadal))
+        .then(() => loginpage.enterPassword(dataJS.passwordUserRafaNadal))
```

Comment:
Every browser.wait and expect().tobe should be present inside corresponding page object.
Page object contains getters and methods. Method can be "ValidateContentFor.." for example.
In test you only call methods from page objects, there should be no clicking, expectation and waits inside test.

Reply:
I removed expectations and waits from test,